prophyc/generators/python.py

```diff
@@ -65,7 +66,7 @@ def translate_include(self, include):
         self.included_symbols.update(included)
         if not included:
             return ""
-        statement_begin = "from %s import " % include.name.split("/")[-1]
+        statement_begin = "from .%s import " % include.name.split("/")[-1]
```

Comment:
That's reasonable, it enables to put codecs into a python package in python3. I see no drawbacks.